frontend/composables/useURLParams.ts

```diff
@@ -49,7 +105,9 @@ export const useURLParams = () => {
   const fromQueryReactiveWithDefault = (key: string, def: string): WritableComputedRef<string> => {
     const fqr = fromQueryReactive(key)
     return computed({
-      get: () => fqr.value ?? def,
+      get: () => {
```

Comment:
Looks like this change can probably be undone

Reply:
GC